.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,38 @@
+name: 🚀 Feature request
+description: Submit a proposal/request for a new TorchGeo feature
+labels: [feature]
+
+body:
+- type: textarea
+  attributes:
+    label: Summary
+    description: >
+      A clear and concise summary of your suggestion.
+- type: textarea
+  attributes:
+    label: Rationale
+    description: >
+      Why is this feature important? Is it related to a problem you are experiencing?
+- type: textarea
+  attributes:
+    label: Implementation
+    description: >
+      If you've thought about how to implement this feature, describe your proposed
+      solution.
+- type: textarea
+  attributes:
+    label: Alternatives
+    description: >
+      Are there any alternatives to the solution you've proposed?
+- type: textarea
+  attributes:
+    label: Additional information
+    description: >
+      Any additional information that might be relevant to the proposed feature.
+- type: markdown
+  attributes:
+    value: >
+      Thanks for suggesting this awesome new feature! TorchGeo is an open-source project
+      maintained by its users. If you're Python savvy and want to contribute a pull
```

Comment:
Kind of sounds like we are cool with taking on any new features. Maybe we can rephrase to include that we'll discuss the feature, then, if it is in scope: forget about it for a few months, discuss again, then implement :)